[2022-12-27T02:40:31.662+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: new_table_and_data_import_dag_v1.create_table manual__2022-12-27T02:40:29.409709+00:00 [queued]>
[2022-12-27T02:40:31.671+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: new_table_and_data_import_dag_v1.create_table manual__2022-12-27T02:40:29.409709+00:00 [queued]>
[2022-12-27T02:40:31.671+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2022-12-27T02:40:31.672+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 1
[2022-12-27T02:40:31.672+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2022-12-27T02:40:31.718+0000] {taskinstance.py:1304} INFO - Executing <Task(PostgresOperator): create_table> on 2022-12-27 02:40:29.409709+00:00
[2022-12-27T02:40:31.722+0000] {standard_task_runner.py:55} INFO - Started process 627 to run task
[2022-12-27T02:40:31.725+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'new_table_and_data_import_dag_v1', 'create_table', 'manual__2022-12-27T02:40:29.409709+00:00', '--job-id', '91', '--raw', '--subdir', 'DAGS_FOLDER/new_table_and_data_import_dag.py', '--cfg-path', '/tmp/tmpezlxhivw']
[2022-12-27T02:40:31.726+0000] {standard_task_runner.py:83} INFO - Job 91: Subtask create_table
[2022-12-27T02:40:31.855+0000] {task_command.py:389} INFO - Running <TaskInstance: new_table_and_data_import_dag_v1.create_table manual__2022-12-27T02:40:29.409709+00:00 [running]> on host 64867ca07077
[2022-12-27T02:40:32.052+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=new_table_and_data_import_dag_v1
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2022-12-27T02:40:29.409709+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-12-27T02:40:29.409709+00:00
[2022-12-27T02:40:32.053+0000] {sql.py:253} INFO - Executing: 
            CREATE TABLE IF NOT EXISTS test_table (
                ticker varchar(3),
                year integer,
                quarter integer,
                invest_cost integer,
                from_invest integer,
                from_financial integer,
                from_sale integer,
                free_cash_flow double precision,
                PRIMARY KEY (ticker, year, quarter),
                FOREIGN KEY (ticker) REFERENCES listing_companies (ticker)
            );
        
[2022-12-27T02:40:32.060+0000] {base.py:73} INFO - Using connection ID 'financial_database' for task execution.
[2022-12-27T02:40:32.390+0000] {base.py:73} INFO - Using connection ID 'financial_database' for task execution.
[2022-12-27T02:40:32.399+0000] {sql.py:364} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS test_table (
                ticker varchar(3),
                year integer,
                quarter integer,
                invest_cost integer,
                from_invest integer,
                from_financial integer,
                from_sale integer,
                free_cash_flow double precision,
                PRIMARY KEY (ticker, year, quarter),
                FOREIGN KEY (ticker) REFERENCES listing_companies (ticker)
            );
        , parameters: None
[2022-12-27T02:40:32.411+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=new_table_and_data_import_dag_v1, task_id=create_table, execution_date=20221227T024029, start_date=20221227T024031, end_date=20221227T024032
[2022-12-27T02:40:32.738+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2022-12-27T02:40:32.909+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
